package.json

```diff
@@ -10,9 +10,11 @@
     "contracts:compile": "yarn contracts:compile:abi && yarn contracts:compile:v3",
     "relay": "relay-compiler relay.config.js",
     "relay-thegraph": "relay-compiler relay_thegraph.config.js",
+    "apollo": "graphql-codegen --config apollo-codegen.ts",
```

Comment:
These should be named `apollo:generate:data` and `apollo:generate:thegraph`. I would also put them directly above `apollo:generate`.